curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f33310c1851c6c4da013250.md

```diff
@@ -7,33 +7,34 @@ dashedName: step-8
 
 # --description--
 
-It's time to add some menu content. Add a `main` element below the existing `header` element. It will eventually contain pricing information about coffee and desserts offered by the cafe.
+Since the `p` element added in the previous step provides supplemental information about the cafe, nest both the `h1` and `p` elements in a `header` element.
```

Comment:
I think the issue was, the `h1` element being outside the `main` element.

Reply:
The issue was that if the `header` is outside of the `main` tag, it becomes a banner element and should not in that case have a `h1` as a banner element should be used for repeating content.

But since the markup in this challenge doesn't require a banner element, moving `header` inside `main` should also resolve this issue as the removes the `header` elements banner status.

Reply:
The original issue is that it is a best practice to have the `h1` element at the beginning of the `main` element and since there is no reason not to do that for this project then we should move it into `main`. We can lose the `header` element completely if we want, since it is now inside of `main` it has no special accessibility qualities (it is basically a `div`). In fact, the updated step 8 in this PR should probably be removed completely since the rational for adding the `header` element no longer exists.

Reply:
I say we go with that, then.

@DerrykBoyd Would you mind removing the `header` completely.

Reply:
If that results in removing a step from the challenge then I think that increases the scope here quite a bit? I'll need to figure out how to do that.

I assumed that the rational for the header in this challenge was to demo the of grouping the `h1` and the `p` element as introductory content.

Reply:
@DerrykBoyd I understand your argument. In some ways it does seem like these two should be grouped together somehow. And there actually is something now that I think would satisfy this, the [`hgroup`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hgroup) element. If I had to choose between the two, I think this would be the preferred way to go over using `header`. @ShaunSHamilton, do we want to introduce the `hgroup` element? It's not one that gets used a lot, and it doesn't really enhance accessibility in any way, but I think it would be preferred over `header`. And to be clear, I don't necessarily have a problem with using `header` here, I just don't think the current explanation in the step for using it is good. And to be honest, unless needed for styling purposes, I don't think we need to wrap these two elements at all, which would be my first choice.

Reply:
I would vote for `header` over `hgroup` at this point in the curriculum. It's more common and easier to understand. I'm fine with removing the header, but I don't think it's necessary either. This PR has had some work done on it and it resolves the issue.

Reply:
I'm happy to work on removing the header element since that seems to be the preferred approach.

It would be nice if that work could be moved to a separate PR if it involves removing a step from a challenge as that's not something I'm familiar with.

Reply:
@bbsmooth Thank you for teaching me about the `hgroup` - it does sound like it was made for this situation.

---

@DerrykBoyd If this PR as is resolves the issue, then I am happy with it going in as is. I do think we should remove the `header` altogether, in another PR.